statusHandler/view/termuic/termuiConsole.go

```diff
@@ -22,7 +22,7 @@ const refreshInterval = time.Second
 // in order to clean the unwanted appeared characters
 const numOfTicksBeforeRedrawing = 10
 
-var log = logger.DefaultLogger()
+var log = logger.GetOrCreate("statushandler/termui")
```

Comment:
why this logger key ? shouldn't we keep the pattern of the directories ?

Reply:
done